backend/implemented/cruds/crud.py

```diff
@@ -0,0 +1,74 @@
+from cassandra.cluster import Cluster
+from cassandra.query import dict_factory
+import ulid
+
+from openapi_server.models.employee import Employee
+
+def get_session(host):
+    cluster = Cluster([host])
+    session = cluster.connect()
+    session.row_factory = dict_factory
+    return session
+
+class CassandraConnector():
+    def __init__(self):
+        self.host = 'db_employee'
+        session = get_session(self.host)
+        # create keyspace
+        session.execute(
+            "CREATE KEYSPACE IF NOT EXISTS employee \
+            WITH REPLICATION = { \
+            'class' : 'SimpleStrategy', 'replication_factor' : '1' \
+            };"
+        )
+        # create table
+        session.execute(
+            "CREATE TABLE IF NOT EXISTS employee.info \
+            (id text PRIMARY KEY, name text, attr text);"
+        )
+
+    def create(self, data):
```

Comment:
For things like these functions, maybe a class named "EmployeeTable" would be nice